menggunakan-git

```diff
@@ -26,12 +26,13 @@ rm :
 Workflow
 
 1. Instalasi
-apt-get install git-svn
+apt-get install git-svn (jika Anda sudah root)
```

Comment:
This could be confusing, if people type it into the command line.  `bash` add `(jika Anda sudah root)` as parameters.

Better to use

```
apt-get install git-svn # jika Anda sudah root
```

or

```
sudo apt-get install git-svn
```